src/main/java/com/shamil/io/FileStorageManager.java

```diff
@@ -34,18 +34,13 @@ public void loadApplicationState(UserRepository userRepository,
                                      ReservationRepository reservationRepository) {
         List<User> users = loadUsers();
         if (users != null) {
-            for (User user : users) {
-                userRepository.addUser(user);
-            }
+            userRepository.addAll(users);
```

Comment:
well, all this stuff shouldn't be here. No saveALl(getAll()) way.  it should be under the surface in repository